### PR TITLE
playing with PR

### PR DESCRIPTION
**Pull request by SeanFeldman**
##Connect to #7

### CODE REVIEW

**Comment by SeanFeldman** [comment]
this is a comment for PR

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # TestRepo
+testing testing testing
```
Comment (SeanFeldman) [inline README.md]: review comment 1

Comment (SeanFeldman) [inline README.md]: starting a review

**Comment by SeanFeldman** [comment]
bla bla

File: README.md (hunk 1 above)
Comment (SeanFeldman) [inline README.md]: let's discuss why we need the repetition.
blah blah blah...

Comment (SeanFeldman) [inline README.md]: this is shit

Comment (SeanFeldman) [inline README.md]: and this is shit

Comment (SeanFeldman) [inline README.md]: but that one is a supurb shaise

Comment (SeanFeldman) [inline README.md]: SF comment 1

Comment (dbelcham) [inline README.md]: db review 1

Comment (dbelcham) [inline README.md]: db review 2

Comment (SeanFeldman) [inline README.md]: SF comment 2

Comment (dbelcham) [inline README.md]: db review 3

Comment (SeanFeldman) [inline README.md]: SF comment 3

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
 # TestRepo
```
Comment (SeanFeldman) [inline README.md]: one two three

Comment (SeanFeldman) [inline README.md]: and more

**Pull request closed without merging**